Gemfile

```diff
@@ -21,6 +21,7 @@ gem "jekyll-theme-modernist", "~> 0.1.1"
 # If you have any plugins, put them here!
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.6"
+  gem "jekyll-paginate-v2"
```

Comment:
Should we constraint the minimum version of this plugin?

Reply:
@msashkin didn't find any mention about version in available examples or plugin doc

Reply:
@msashkin didn't find any mention about version in available examples or plugin doc